sql/core/src/test/scala/org/apache/spark/sql/execution/WholeStageCodegenSuite.scala

```diff
@@ -232,6 +232,93 @@ class WholeStageCodegenSuite extends QueryTest with SharedSparkSession
     }
   }
 
+
+  test("SPARK-44060 Code-gen for build side outer shuffled hash join") {
```

Comment:
Shall we have the following to make this test case explicit in term of the test requirement? 
```
withSQLConf(SQLConf.ENABLE_BUILD_SIDE_OUTER_SHUFFLED_HASH_JOIN_CODEGEN.key -> "true") {
```

Reply:
Sure, will do!